aggregator/src/datastore.rs

```diff
@@ -5189,53 +5189,124 @@ pub mod test_util {
     use deadpool_postgres::{Manager, Pool};
     use janus_core::time::Clock;
     use lazy_static::lazy_static;
-    use rand::{distributions::Standard, thread_rng, Rng};
+    use rand::{distributions::Standard, random, thread_rng, Rng};
     use ring::aead::{LessSafeKey, UnboundKey, AES_128_GCM};
     use std::{
         env::{self, VarError},
+        mem::take,
         process::Command,
         str::FromStr,
+        sync::{Arc, Barrier, Weak},
+        thread::{self, JoinHandle},
     };
-    use testcontainers::{images::postgres::Postgres, Container, RunnableImage};
-    use tokio_postgres::{Config, NoTls};
+    use testcontainers::{images::postgres::Postgres, RunnableImage};
+    use tokio::sync::{oneshot, Mutex};
+    use tokio_postgres::{connect, Config, NoTls};
     use tracing::trace;
 
-    /// The Janus database schema.
-    pub const SCHEMA: &str = include_str!("../../db/schema.sql");
+    struct EphemeralDatabase {
```

Comment:
Since this commit touches a lot of `mode datastore::test_util`, it might be a good time to move it into a new module like `aggregator/src/datastore/test_util.rs` to make it easier to navigate this large file but I'm OK with this as-is.